core/tests/puppeteer-acceptance-tests/utilities/user/logged-in-user.ts

```diff
@@ -225,7 +224,8 @@ const emptySuggestedForYouSectionSelector =
 const learnerGreetingsSelector = '.e2e-test-learner-greetings';
 const addGoalsButtonInRedesignedLearnerDashboard = '.e2e-test-add-goals-button';
 const newGoalsListInRedesignedLearnerDashboard = '.e2e-test-new-goals-list';
-const goalCheckboxInRedesignedLearnerDashboard = `${newGoalsListInRedesignedLearnerDashboard} mat-checkbox`;
+const goalCheckboxInRedesignedLearnerDashboard =
+  '.oppia-learner-dash-goals-checkbox';
```

Comment:
Why change from `e2e-test-*` class to `oppia-learner-*` class. We prefer `e2e-test-*`.